.github/workflows/tests.yml

```diff
@@ -43,6 +43,10 @@ jobs:
         run: |
           cargo make --version || cargo install cargo-make
       - name: run test
+        env:
```

Comment:
`lookup_keccak` involving some deep recursion when build the circuit. I think it's a existing issue, just the new change hit the threshold unluckily. With that, we temporarily increase `RUST_MIN_STACK` to make tests pass. It doesn't break functional correctness.